internal/config/projectconfig/init.go

```diff
@@ -59,3 +44,33 @@ func Init(dir string, projectName string, projectContext *ZeroProjectConfig) {
 		exit.Fatal(fmt.Sprintf("Failed to create config file %s", constants.ZeroProjectYml))
 	}
 }
+
+func GetProjectFileContent(projectConfig ZeroProjectConfig) string {
+	var tpl bytes.Buffer
+	tmpl := template.New("sa")
+	tmpl, err := tmpl.Parse(exampleConfig)
+	if err != nil {
+		exit.Fatal(fmt.Sprintf("Failed to parse the sample Zero module config file %s", constants.ZeroProjectYml))
+	}
+
+	type tempProjectConfig struct {
```

Comment:
You can combine the definition and instantiation here by having an unnamed struct like this:
`t := struct {
   Name string 
    ... 
}{
    projectConfig.Name
    ...
}
`